python/py-presets/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        bmcfee presets 0.1.3
+revision            0
+name                py-${github.project}
+categories-append   devel
+
+license             ISC
+maintainers         nomaintainer
+platforms           darwin
+
+description         A python module to manipulate default parameters\
+                    of a module's functions.
+
+long_description    ${description}
+
+checksums           rmd160  c8081e31934f9ea973524e5ea0bb8e5fe55a7b25 \
+                    sha256  a3afc2c2bbc9c7ca87fc2be4b2b11081e07e9caec2e1ee09a3e5952f33874a8f \
+                    size    10684
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-numpydoc
+
+    depends_test-append \
```

Comment:
Probably only pytest itself is required for testing.

Reply:
The requirements say it wants `pytest-cov`, so this is a copy-and-paste, but I’ll do as you prefer.

Reply:
Done thanks.